webpack.config.js

```diff
@@ -34,9 +34,6 @@ module.exports = async (env, options) => {
             exclude: /node_modules/,
             use: {
               loader: "babel-loader",
```

Comment:
Comma should be deleted too. 

```suggestion
              loader: "babel-loader"
```

Reply:
Actually, prettier is setup to ask for commas to be there.  You'll notice that is the case in the rest of the file as well.  This seems to be the general practice except in the case of JSON which explicitly doesn't accept this.